.gitignore

```diff
@@ -11,3 +11,4 @@ target
 Cargo.lock
 Cargo.toml
 !library/crates/Cargo.toml
+!library/crates/Cargo.lock
```

Comment:
Crates-universe depends on this Cargo.lock to be present.

Reply:
Why? How was it working before we had this?

Reply:
`.gitignore` only affects files that aren't tracked.

Reply:
So it was working because we'd generated `Cargo.lock` files on our local machines and that was allowing the update script to work?

Reply:
Because it was already present and tracked, modifications would be detected by git. 

I discovered that this was a problem when I removed Cargo.lock _through IDEA's project tree view,_ which runs `git rm ...`. `git` then promptly ignored the newly generated lockfile, because it was no longer tracked.